source/develop/release-management/features/storage/glance-integration.html.md

```diff
@@ -29,40 +33,42 @@ This feature will allow oVirt users to consume (use), export and share images wi
 
 ### VDSM
 
-*   Implement the imageImport and imageExport commands, either explicitly supporting glance or with a more generic http repository support (with additional special headers in the glance case)
+*   Implement the `imageImport` and `imageExport` commands, either explicitly supporting glance or with a more generic http repository support (with additional special headers in the glance case)
 
 ## Detailed Description
 
-The Glance project provides services for discovering, registering, and retrieving virtual machine images. These images are to be intended as oVirt Templates.
+The Glance project provides services for discovering, registering, and retrieving virtual machine images. These images are to be intended as oVirt templates.
 
 ### Phase 1
 
-*   Add the new Storage Domain for Glance (identified by the StorageType GLANCE and the StorageDomainType Image)
+*   Add the new Storage Domain for Glance (identified by the `StorageType#GLANCE` and the `StorageDomainType#Image`)
 *   Add, edit and remove the Glance Storage Domains
 *   List the available images in the Glance Storage Domains (leveraging the current infrastructure that oVirt uses for NFS ISO domains, which have many aspects in common)
 *   Import images from Glance as oVirt Templates or floating Disks
 *   Export single (floating) disks (without snapshots) to Glance
 
 ## VDSM API
 
-      importImage(spUUID, sdUUID, imgUUID, volUUID, importInfo)
-      exportImage(spUUID, sdUUID, imgUUID, volUUID, exportInfo)
+    importImage(spUUID, sdUUID, imgUUID, volUUID, importInfo)
+    exportImage(spUUID, sdUUID, imgUUID, volUUID, exportInfo)
 
-      importInfo = {
-         'method': 'http',
-         'url': '`[`http://glance/v1/images/image-uuid`](http://glance/v1/images/image-uuid)`',
-         'headers': {
-           'X-Auth-Token': 'token-uuid',
-         },
-       }
+    importInfo = {
+       'method': 'http',
+       'url': '`[`http://glance/v1/images/image-uuid`](http://glance/v1/images/image-uuid)`',
+       'headers': {
+         'X-Auth-Token': 'token-uuid',
+       },
+    }
 
 ## Future Phases
 
 Import and Export entire VMs with multiple disks and snapshots using the OVA container format.
 
 ## User Experience
 
-![ thumb | left](NewGlanceDomain1.png  "fig: thumb | left") ![ thumb | left](GlanceDomainListing1.png  "fig: thumb | left")
+![New Glace Domain Dialog](images/wiki/NewGlanceDomain1.png)
```

Comment:
@danielerez I don't understand this comment. The files are there, but the image links were broken on the page. One of the things this patch does is fix them. Spin up a server (using `./run-server.sh` and you can see for yourself).